.golangci.yml

```diff
@@ -35,6 +35,8 @@ linters:
 linters-settings:
   misspell:
     locale: US
+  goimports:
+    local-prefixes: github.com/mongodb/mongodb-atlas-kubernetes
```

Comment:
Corresponding change in IDEs:
- VS Code:
```yaml
"gopls": {
    "formatting.local": "github.com/mongodb/mongodb-atlas-kubernetes/"
}
```
or
```yaml
"go.formatFlags": [
    "-local=github.com/mongodb/mongodb-atlas-kubernetes"
]
```
- GoLand: <details>
	  <summary>(Screenshot)</summary>
	  ![image](https://user-images.githubusercontent.com/1939685/106726994-10afb480-661c-11eb-8851-5d32401771f2.png)
	  </details>

Reply:
just fyi: have checked and the pre-commit hook linter handles this config perfectly, so IDE integration is optional - it will be anyway formated on commit.

Reply:
I actually did not set this up on my WSL machine yet, as pre-commit is failing with a python error...

Reply:
@antonlisovenko Also - I thought `pre-commit` will only diagnose code, but not change it?

Reply:
no, it calls the `golangci-lint`: https://github.com/golangci/golangci-lint/blob/master/.pre-commit-hooks.yaml so performs the updates.